apps/system/js/remote/remote_app_window_manager.js

```diff
@@ -93,6 +112,37 @@
 
     _handle_mozbrowserclose: function() {
       this.killCurrentApp();
+    },
+
+    _handle_mozbrowserloadend: function() {
+      this._sendPresentationLaunched();
+    },
+
+    _sendPresentationDenied: function(id) {
```

Comment:
I would prefer a general function (e.g. `_sendPresentationContentEvent(type, detail)`) but it's up to you.